codeful-logicapp-sample/test.code-workspace

```diff
@@ -0,0 +1,17 @@
+{
+  "folders": [
+    {
+      "name": "test-app",
+      "path": "./test-app"
+    }
+  ],
+  "settings": {
+    "terminal.integrated.env.windows": {
+      "PATH": "C:\\Users\\nikhilsira\\.azurelogicapps\\dependencies\\DotNetSDK;${env:PATH}"
```

Comment:
lets remove all PII from the code.. stuff that is generated by the VS extension, build etc., we don't need to check that in..